### PR TITLE
[ADD] Ticket creation from portal

### PR DESCRIPTION
**Pull request by samuel-domatix**
- Added 2 methods to manage the creation of tickets
- Added a creation form in the portal to enter the ticket data to be created
- Added two buttons to access the ticket creation form. One on the personal ticket list and another on the _portal_my_home_ page

### CODE REVIEW

File: helpdesk_mgmt/controllers/main.py [hunk 1]
```diff
@@ -20,3 +22,47 @@ def support_ticket_close(self, **kw):
...
+                sudo().search([('name', '=', 'Web')]).id,
+            'partner_id':
+                request.env['res.partner'].sudo().search([
+                    ('name', '=', kw['name']),
+                    ('email', '=', kw['email'])]).id
+        }
+        new_ticket = request.env['helpdesk.ticket'].sudo().create(
+            vals)
+        new_ticket.message_subscribe_users(user_ids=request.env.user.id)
+        if kw['attachment']:
```
Comment (pedrobaeza) [inline helpdesk_mgmt/controllers/main.py]:
```suggestion
        if kw.get('attachment'):
```

**Comment by emagdalenaC2i** [comment]
@SamuelDomatix if this PR replace #70 #71 and #72 please close those PR

@alvaroDomatix I think this PR include the main parts of #33 so you can close that PR too

**Review by emagdalenaC2i — APPROVED** [review]
Tested in runbot

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Can be forward-port it to v12 (respecting commit attribution, please).

**Pull request merged**